README.md

```diff
@@ -104,6 +111,21 @@ spec:
           }
         ]
       }
+  secondaryNetwork:
+    cniPlugin:
```

Comment:
@moshe010 do you think we might need to support multiple networking plugins in the same deployment ?

to support multiple, this field would need to be a list.

Reply:
So, based on other discussions as we are going to deploy a set of CNI plugins (containernetowking-plugins)
this should be named imo `cniPlugins`

Reply:
changed